src/Microsoft.DotNet.ProjectModel/Files/IncludeContext.cs

```diff
@@ -38,7 +38,15 @@ public class IncludeContext
             SourceBasePath = sourceBasePath;
             Option = option;
             var token = rawObject.Value<JToken>(option);
-            if (token.Type != JTokenType.Object)
+
+            if (token == null)
+            {
+                IncludePatterns = new List<string>();
+                ExcludePatterns = new List<string>();
+                IncludeFiles = new List<string>();
+                ExcludeFiles = new List<string>();
```

Comment:
When `token.Type != JTokenType.Object`, ExcludePatterns, IncludeFiles, and ExcludeFiles aren't getting set either.  Do you have code that is expecting all these collections to be non-null?  Will you NullRef in that case?